### DISCUSSION TITLE
Unable to list workspace or create connexion using self hosted airbyte python sdk

### DISCUSSION THREAD

**Discussion by Abdi-Arslene** [airbytehq/airbyte#45182] [Source Python CDK] [answered]
Hello airbyte team,
 
I’m working with the Airbyte Python SDK and have a self-hosted Airbyte setup running locally on my machine using abctl. Does the Python SDK support this setup? If not, can you guide me on how to use the Airbyte API with a local installation? Also, where can I find relevant documentation?
this is what i did but i always get errors: 
![image](https://github.com/user-attachments/assets/dff43950-5a05-4952-bc87-77d5b5838c1f)

Thanks in advance!

**Comment by szemek** [comment, marked as answer]
https://reference.airbyte.com/reference/standalone-server-deprecation-and-migration-to-airbyte-server

Instead of `http://localhost:8000/api/v1` use `http://localhost:8000/api/public/v1`

**Reply by Abdi-Arslene** [reply]
Thank you ! Now it works :)
